app/main/views/search.py

```diff
@@ -118,10 +118,11 @@ def delete_service(index_name, doc_type, service_id):
 
 
 def api_response(data, status_code, key='message'):
+    if status_code == 'N/A':
```

Comment:
Is that what the status code comes through as? How strange, wouldn't have expected that.

Instead of checking if it explicitely is the string `N/A`, can we instead check that it is a number/integer? Or use a `try` block and catch the `TypeError`?

Reply:
It might be better to check whether `data` is a `NewConnectionError`; slightly more apparent what's going on than checking whether the status_code is N/A.

Reply:
Hey guys, added a docstring explaining the method, should be clearer now, agree that the 'N/A' thing is weird. 

I've used a Try Except around the if statement now because of the whole 'ask forgiveness not permission' thing.
As this method is used for so many of our requests even a slight performance enhancement here is a good thing.